tests/manage/pv_services/test_verify_rwo_using_multiple_pods.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.helpers import wait_for_resource_state
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.parametrize(
+    argnames='interface',
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL],
+            marks=pytest.mark.polarion_id("OCS-1177")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM],
+            marks=pytest.mark.polarion_id("OCS-1179")
+        )
+    ]
+)
+class TestRwoUsingMultiplePods(ManageTest):
+    """
+    This test class consists of tests to verify RWO access mode by using
+    RWO PVC in multiple pods
+    """
+    @pytest.fixture(autouse=True)
+    def setup(self, interface, pvc_factory, pod_factory):
+        """
+        Create pvc and pod
+        """
+        # Create a PVC
+        self.pvc_obj = pvc_factory(
+            interface=interface,
+            project=None,
+            storageclass=None,
+            size=5,
+            access_mode=constants.ACCESS_MODE_RWO,
+            custom_data=None,
+            status=constants.STATUS_BOUND
+        )
+
+        # Create a pod
+        self.pod_obj = pod_factory(
+            interface=interface,
+            pvc=self.pvc_obj,
+            custom_data=None,
+            status=constants.STATUS_RUNNING
+        )
+
+    def test_verify_rwo_using_multiple_pods(self, interface, pod_factory):
+        """
+        This test case verifies RWO access mode by trying to mount same RWO
+        PVC on different pods, delete running pods one by one and ensure that
+        only one pod is running at a time
+        """
+        pod_objs_running = [self.pod_obj]
+        pod_objs_not_running = []
+
+        # Create 5 new pods using same PVC
+        for _ in range(5):
+            pod_obj = pod_factory(
+                interface=interface,
+                pvc=self.pvc_obj,
+                custom_data=None,
+                status=""
+            )
+            pod_objs_not_running.append(pod_obj)
+
+        # Check the status of pods, delete running pod and verify new pod is
+        # coming up running
+        while pod_objs_running:
+            pod_running_node = pod_objs_running[0].get()['spec']['nodeName']
+
+            # Verify status of pods
+            log.info("Check the status of pods")
+            for pod_obj in pod_objs_not_running:
+                try:
+                    wait_for_resource_state(
```

Comment:
I think pod_obj.reload() is missing

Reply:
reload() is not done because I don't see that is important here. The updated values are not used in later part of this test. Only pod.name is used, which is already available.